sql/migrations/20211122153748_world.sql

```diff
@@ -0,0 +1,136 @@
+
+-- Creature 3599
+UPDATE `gossip_menu` SET `text_id` = 934 WHERE `entry` = 436 AND `text_id` = 4794;
+DELETE FROM `gossip_menu_option` WHERE `menu_id` = 436 AND `id` = 1;
+
+-- Creature 4163
+UPDATE `gossip_menu` SET `text_id` = 4795 WHERE `entry` = 4576 AND `text_id` = 4794;
+
+-- Creature 4214
+UPDATE `creature_template` SET `gossip_menu_id` = 4577 WHERE `entry` = 4214;
+INSERT INTO `gossip_menu` (`entry`, `text_id`, `script_id`, `condition_id`) VALUES
+(4577, 4793, 0, 101),
+(4577, 4795, 0, 100),
+(4577, 5996, 0, 198),
+(4577, 8255, 0, 461),
+(4577, 8283, 0, 459),
+(4577, 8285, 0, 454),
+(4577, 8291, 0, 458),
+(4577, 8296, 0, 460),
+(4577, 8298, 0, 455);
+
+INSERT INTO `gossip_menu_option` (`menu_id`, `id`, `option_icon`, `option_text`, `option_broadcast_text`, `option_id`, `npc_option_npcflag`, `action_menu_id`, `action_poi_id`, `action_script_id`, `box_coded`, `box_money`, `box_text`, `box_broadcast_text`, `condition_id`) VALUES
+(4577, 0, 3, 'I seek training.', 7478, 5, 16, 0, 0, 0, 0, 0, '', 0, 0),
+(4577, 1, 0, 'I wish to unlearn my talents.', 8271, 1, 1, 4461, 0, 0, 0, 0, '', 0, 136),
+(4577, 4, 0, '<Take the letter>', 8548, 1, 1, -1, 0, 16, 0, 0, '', 0, 198),
+(4577, 9, 0, 'Here, I\'d like to give you this token of my love.', 11723, 1, 1, -1, 0, 410, 0, 0, NULL, 0, 462),
+(4577, 10, 0, 'Here, I\'d like to give you this token of my love.', 11723, 1, 1, -1, 0, 411, 0, 0, NULL, 0, 461);
+
+-- Creature 4215
+UPDATE `creature_template` SET `gossip_menu_id` = 4575 WHERE `entry` = 4215;
+INSERT INTO `gossip_menu` (`entry`, `text_id`, `script_id`, `condition_id`) VALUES
+(4575, 4793, 0, 101),
+(4575, 4795, 0, 100),
+(4575, 5996, 0, 198),
+(4575, 8255, 0, 461),
+(4575, 8283, 0, 459),
+(4575, 8285, 0, 454),
+(4575, 8291, 0, 458),
+(4575, 8296, 0, 460),
+(4575, 8298, 0, 455);
+
+INSERT INTO `gossip_menu_option` (`menu_id`, `id`, `option_icon`, `option_text`, `option_broadcast_text`, `option_id`, `npc_option_npcflag`, `action_menu_id`, `action_poi_id`, `action_script_id`, `box_coded`, `box_money`, `box_text`, `box_broadcast_text`, `condition_id`) VALUES
+(4575, 0, 3, 'I seek training.', 7478, 5, 16, 0, 0, 0, 0, 0, '', 0, 0),
+(4575, 1, 0, 'I wish to unlearn my talents.', 8271, 1, 1, 4461, 0, 0, 0, 0, '', 0, 136),
+(4575, 4, 0, '<Take the letter>', 8548, 1, 1, -1, 0, 16, 0, 0, '', 0, 198),
+(4575, 9, 0, 'Here, I\'d like to give you this token of my love.', 11723, 1, 1, -1, 0, 410, 0, 0, NULL, 0, 462),
+(4575, 10, 0, 'Here, I\'d like to give you this token of my love.', 11723, 1, 1, -1, 0, 411, 0, 0, NULL, 0, 461);
+
+-- Creature 4582
+UPDATE `gossip_menu` SET `text_id` = 5993 WHERE `entry` = 4542 AND `text_id` = 5996;
+UPDATE `gossip_menu_option` SET `option_broadcast_text` = 7563, `option_text` = 'I would like to train.' WHERE `menu_id` = 4542 AND `id` = 0;
+
+-- Creature 4584
+UPDATE `gossip_menu_option` SET `option_broadcast_text` = 7563, `option_text` = 'I would like to train.' WHERE `menu_id` = 4541 AND `id` = 0;

```

Comment:
Broadcast Text ID 7563 has an emote assigned, so I think this text is used in npc_text, and for the gossip option you should use 2548.

Reply:
Not sure what you mean

Reply:
Look at broadcast text id 7563. It has emote_id1 = 1. Blizzard only assigns emotes to broadcast texts that are used in npc_text. So i suggest yo should assign broadcast id 2548 instead to this gossip option, which is the exact same text, but with no emote (gossip option cannot have emote, its clicked by player).